core/base/src/TApplication.cxx

```diff
@@ -1339,6 +1339,7 @@ void TApplication::Help(const char *line)
          Printf("   Ctrl+D           : quit ROOT (if empty line)");
          Printf("   Ctrl+C           : send kSigInt interrupt signal");
          Printf("   Ctrl+Z           : send kSigStop pause job signal");
+         Printf("   Ctrl+\\           : send kSigQuit quit job signal");
```

Comment:
If we advertise this shortcut we should either support it on Windows as well or (less ideal) hide this line on Windows.

Reply:
I've proposed an idea, but maybe @bellenot can advice on whether this is fine or it's better to call ::abort()

Reply:
At first glance I have no strong opinion. One can also simply use non Windows specific method like `std::abort`, `std::exit`, or `std::terminate`